tests/acceptance/features/apiSharingNg/listPermissions.feature

```diff
@@ -1300,4 +1300,44 @@ Feature: List a sharing permissions
           }
         }
       }
+      """
+
+
+  Scenario: try to list the permissions of a user's personal space by a different member
+    Given using spaces DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    When user "Brian" tries to list the permissions of space "Personal" owned by "Alice" using permissions endpoint of the Graph API
+    Then the HTTP status code should be "404"
```

Comment:
should not it be a 403 status code? user accessing the another owned resource? just a guess.

Reply:
IMO, `404` is good here. because no user with any role should not be able to find/access other user's personal space